File: Structy_2/2_Arrays_Strings/Uncompress/uncompress.py
def uncompress(s):
  numbers = '01234567890'
  result = []
  i = 0
  j = 0
  while j < len(s):
    if s[j] in numbers:
      j += 1
    else:
      num = int(s[i:j])
      result.append(s[j] * num)
      j += 1
      i = j
  
  return "".join(result)

File: Structy_2/2_Arrays_Strings/1_Uncompress/uncompress.js
const uncompress = (s) => {
  // todo
  let result = [];
  let nums = '0123456789'
  
  let i = 0;
  let j = 0;
  while(j < s.length){
    if(nums.includes(s[j])){
      j +=1
    } else{
      const num = Number(s.slice(i,j));
      for (let count = 0; count < num; count += 1){
        result.push(s[j])
      }
      j+=1;
      i = j;
    }
  }
  return result.join('');
};